internal/ionoscloud/ionoscloud.go

```diff
@@ -0,0 +1,62 @@
+package ionoscloud
+
+import (
```

Comment:
Should we push cloud provider work in progress on a public branch?

Reply:
good point. FMPOV having feature incomplete code in main branch is acceptable ( as long as it does not break things). As long as we not create a release this code is not used.  Feature complete PR can become big and tend to stay open for long time, which can have a negative impact on the  review process and rebasing/merging. One answer for this problem would be introducing [a gitflow branching model](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow). But imho we should try to stay on a simple model as long as we don't have "real" problems. @mspoeri do you have a opinion  about that?

Reply:
I see this the same way. Yes, GitFlow would be an answer but a very long one for a - at least in our case - very simple question. As stated for sure we should enforce that the main branch always is functional with what is there but if a feature ain't completely implemented that way, that we don't want to create a release this should not bother us.

Hard time for people using the "latest" tag for their dependency but maybe they someday learn a lesson 😉